### DISCUSSION TITLE
Release Health: Retiring Session Duration - Your Feedback Needed

### DISCUSSION THREAD

**Discussion by ale-cota** [getsentry/sentry#42716] [Upcoming Changes]
Hi everyone,

A new year is upon us and here at Sentry we’re pretty excited looking ahead at all the awesome stuff we want to build. But it’s also a good opportunity to stop and have a critical look at features we’ve already shipped and evaluate whether they’re serving their purpose and bringing value to most of our customers.

The **session duration** metric part of our Release Health product is one of the things we’ve had a look at. It’s one of the metrics that contributes most significantly to our dataset growth, however only around 0.3% of our users visiting the Release Details page actually check it. Removing this metric will allow us to add more health diagnostics such as ANR (App Not Responding) rate sooner, without the need to further scale our infrastructure. 

<img width="911" alt="Screenshot 2023-01-03 at 17 19 02" src="https://user-images.githubusercontent.com/26696515/210397399-a934b52c-8357-4b2b-a42c-ec3e49d3f779.png">

If you’re one of our Release Health users who regularly verify the **session duration** metric, we’d like to hear more about why this is valuable to you and how you use it. Your feedback will help us make a decision on how to proceed. 

If we do proceed to remove the feature, we will provide timely communication and try to suggest other ways in which you can maybe achieve similar outcomes in Sentry.

Thanks and looking forward to your comments!

**Comment by therealarkin** [comment]
Excited to see spring cleaning coming!

**Comment by ale-cota** [comment]
We've decided to proceed with removing the session duration metric this week. Everyone stay tuned for the ANR (App Not Responding) rate coming soon to Release Health :)

**Comment by jskrepnek** [comment]
An alternative to removing it could have been to offer it as an opt-in at the client config level and / or to use a sample rate.

I had never used it previously but coincidentally got asked to pull it for the first time just after this went down.  Bad timing.  Is there any way to access historical data?
